include/functions.h

```diff
@@ -255,6 +255,15 @@ void show_damage_popup(f32 x, f32 y, f32 z, s32 damageAmount);
 void func_80070B50(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4);
 void func_800718D0(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, s32 arg5);
 void func_80070BB0(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, s32 arg5);
+void func_80071FF0(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, s32 arg5);
+void func_80070970(s32 arg0, f32 arg1, f32 arg2, f32 arg3, s32 arg4, s32 arg5);
+void func_800709D0(s32 arg0, f32 arg1, f32 arg2, f32 arg3, f32 arg4, s32 arg5);
+void func_80070130(s32 arg0, f32 arg1, f32 arg2, f32 arg3);
+void func_800701F0(s32 arg0, f32 arg1, f32 arg2, f32 arg3);
+void add_xz_vec3f(Vec3f* vector, f32 speed, f32 angleDeg);
+void play_movement_dust_effects(s32 var0, f32 xPos, f32 yPos, f32 zPos, f32 angleDeg);
+
+Actor func_80072110(s32, s32, s32, s32, f32, s32); // Probably wrong
```

Comment:
This was from a Function that I stopped working on (for now at least) and no matter what it returned, it didn't seem to match. it normally has to be `Actor*`, I agree with that, but it somehow broke everything.
For now this is just there so I don't forget about it, but I can also just remove it since it's wrong.

Reply:
yeah remove it then